scripts/installCommonDeps.sh

```diff
@@ -299,13 +299,30 @@ install_licode(){
 }
 
 install_quic(){
+  # QUIC IO
   rm $ROOT/third_party/quic-lib -rf
   mkdir $ROOT/third_party/quic-lib
 
   pushd ${ROOT}/third_party/quic-lib
   wget https://github.com/open-webrtc-toolkit/owt-deps-quic/releases/download/v0.1/dist.tgz
   tar xzf dist.tgz
   popd
+
+  # QUIC transport
+  local QUIC_SDK_VERSION=`cat $ROOT/source/agent/addons/quic/quic_sdk_version`
+  local QUIC_TRANSPORT_PATH=$ROOT/third_party/quic-transport
+  rm -r $QUIC_TRANSPORT_PATH
```

Comment:
{} around the var name.  Also do precheck when running rm -r for path validity.

Reply:
Fixed.